assets/src/block-validation/icon.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import AMPToolbarIcon from '../../images/amp-icon-toolbar.svg';
+import AMPToolbarIconBroken from '../../images/amp-toolbar-icon-broken.svg';
+import AMPNewTabIcon from '../../images/amp-new-tab-icon.svg';
+
+/**
+ * Plugin icon.
+ *
+ * @param {Object} props
+ * @param {boolean} props.hasBadge Whether the icon is showing a number.
+ */
+function IconSVG( { hasBadge } ) {
+	return (
+		<span className={ `amp-toolbar-icon${ hasBadge ? ' amp-toolbar-icon--has-badge' : '' }` }>
+			<AMPToolbarIcon />
+		</span>
+	);
+}
+IconSVG.propTypes = {
+	hasBadge: PropTypes.bool.isRequired,
+};
+
+/**
+ * Plugin icon when AMP is broken at the URL.
+ *
+ * @param {Object} props
+ * @param {boolean} props.hasBadge Whether the icon is showing a number.
+ */
+export function BrokenIconSVG( { hasBadge } ) {
```

Comment:
Seems this doesn't need to be exported.

```suggestion
function BrokenIconSVG( { hasBadge } ) {
```